### PR TITLE
Bug BL-491/resource field and labels

### PR DESCRIPTION
**Pull request by miguelaledesma**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Created an inline style for each of the labels on the resource page in the dashboard. They are mobile friendly. 

Fixes # (issue)

## Loom Video

(https://www.loom.com/share/a712e0d9b9ee4728922b3bbcd71373d9)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by attaris978 — COMMENTED** [review]
1) As these all share the same styling, they can be combined into one block.
2) Be very careful of using !important. If the next Labs participant comes in and wants to style that particular element, they may be SOL without first undoing your code and then implementing their own. It's like being in a room and wanting a little fresh air, but instead of opening the window you smash it with a sledge hammer. Yes, you're now enjoying that fresh air that you want, but when the next person comes in they won't be able to just crack the window a bit.

**Review by CodyAskew9 — COMMENTED** [review]
Small things make big differences good work

File: src/components/common/styles/Resources.css [hunk 1]
```diff
@@ -28,3 +28,18 @@
   justify-content: space-between;
   padding: 4% 0;
 }
+
+[title="Resource"] {
+  color: black !important;
```
Comment (ElijahHopkin) [inline src/components/common/styles/Resources.css:33]: I agree with Ryan--who commented earlier. I think there's a better way to set this CSS than using `!important`. It's definitely a cleaner result, but I think using this solution will lead to problems down the road. I would recommend inline styling or playing with this CSS file to get it to stick without using !important.

**Review by ElijahHopkin — CHANGES_REQUESTED** [review]
CSS is made through brute force, and will likely create problems for some future developer because they may not be aware that we used ' !important'
Use some other means so that future development is smoother.

**Review by jettyblue — DISMISSED** [review]
Excellent work here and definitely fixes the issue of dark mode font color!

**Review by CodyAskew9 — APPROVED** [review]

File: src/components/common/styles/Resources.css (hunk 1 above)
Comment (miguelaledesma) [reply src/components/common/styles/Resources.css]: Fixed these changes you requested!

**Review by ElijahHopkin — APPROVED** [review]

File: src/components/common/styles/Resources.css (hunk 1 above)
Comment (ElijahHopkin) [reply src/components/common/styles/Resources.css]: I like this a lot. :)
Don't know if you need to record a new loom video to show these new changes or not, but I think this approach is much more manageable down the road.

**Review by ethanrmiles — APPROVED** [review]
Definitely see what you were going for with `!important`, but believe your refactoring has accomplished the same thing in a more streamlined way. Good work Miguel!

**Pull request merged**